integration-tests/src/test/java/oracle/kubernetes/operator/JrfInOperator.java

```diff
@@ -0,0 +1,179 @@
+// Copyright 2019, Oracle Corporation and/or its affiliates.  All rights reserved.
+//   Licensed under the Universal Permissive License v 1.0 as shown at
+//   http://oss.oracle.com/licenses/upl.
+
+package oracle.kubernetes.operator;
+
+import oracle.kubernetes.operator.utils.ExecCommand;
+import oracle.kubernetes.operator.utils.ExecResult;
+import oracle.kubernetes.operator.utils.JRFDomain;
+import oracle.kubernetes.operator.utils.Operator;
+import oracle.kubernetes.operator.utils.OracleDB;
+import oracle.kubernetes.operator.utils.TestUtils;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.FixMethodOrder;
+import org.junit.Test;
+import org.junit.runners.MethodSorters;
+
+/**
+ * Simple JUnit test file used for testing Operator for JRF domains.
+ *
+ * <p>This test is used for creating Operator(s) and multiple JRF domains which are managed by the
+ * Operator(s).
+ */
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
+public class JrfInOperator extends BaseTest {
```

Comment:
I think you have not named the test with IT* as it will get called in the regular integration tests. But can you name the class as JrfInOperatorTest as its a test class.

Reply:
Yes, that's true. I will change it to JrfInOperatorTest. Thanks.

Reply:
@vanajamukkara we may need to re-think about the name change. The maven-surefire-plugin will automatically run the tests cases in **/*Test.java. If we changed the filename to JrfInOperatorTest.java, we need to configure maven-surefire-plugin to skip the tests in all profiles including java-integration-tests, otherwise the jrf tests will run automatically.